### PR TITLE
Scheduler Add support to frequency type:

### PR DESCRIPTION
**Pull request by yamelsenih**
This pull request add support to the follows frequencies to Scheduler
- Second
- Does Not Repeat
- Weekly
- Monthly
- Yearly
Also allows define custom Date Next Run for a Schedule

A example for only a run of schedule
Screenshot_20210102_001441

### CODE REVIEW

File: serverRoot/src/main/server/org/compiere/server/AdempiereServer.java [hunk 1]
```diff
@@ -198,7 +201,19 @@ public void runNow()
 		//
 		log.fine(getName() + ": " + getStatistics());
 	}	//	runNow
-
+
+	/**
+	 * Validate if is fixed time
+	 * @return
+	 */
+	private boolean isValidForRun() {
```
Comment (mckayERP) [inline serverRoot/src/main/server/org/compiere/server/AdempiereServer.java:209]: For testing - if the junit upgrades gets approved - consider making this a protected method.  It is difficult to test private methods.

File: serverRoot/src/main/server/org/compiere/server/AdempiereServerMgr.java [hunk 2]
```diff
@@ -557,7 +564,11 @@ public String getServerCount()
 	 */
 	public Timestamp getStartTime()
 	{
-		return m_start;
+		return startTime;
 	}	//	getStartTime
 
+	public static void main(String[] args) {
```
Comment (mckayERP) [inline serverRoot/src/main/server/org/compiere/server/AdempiereServerMgr.java:570]: Why a main method?

**Review by mckayERP — COMMENTED** [review]
OK - but why add the main() method?

File: serverRoot/src/main/server/org/compiere/server/AdempiereServer.java (hunk 1 above)
Comment (yamelsenih) [reply serverRoot/src/main/server/org/compiere/server/AdempiereServer.java:209]: You are right, I will go

File: serverRoot/src/main/server/org/compiere/server/AdempiereServerMgr.java (hunk 2 above)
Comment (yamelsenih) [reply serverRoot/src/main/server/org/compiere/server/AdempiereServerMgr.java:570]: Just go to run with current IDE. Maybe can be removed

**Pull request merged**